v21.2/cockroach-debug-job-trace.md

```diff
@@ -0,0 +1,74 @@
+---
+title: cockroach debug job-trace
+summary: Learn the commands for collecting debug information from all nodes in your cluster.
+toc: true
+---
+
+{{site.data.alerts.callout_danger}}
+We strongly recommend only using `cockroach debug job-trace` when working directly with the Cockroach Labs support team.
+{{site.data.alerts.end}}
+
+The [`cockroach debug job-trace`](cockroach-commands.html) command connects to your cluster and collects trace payloads for a running job. The trace payloads are helpful for debugging why a job is not running as expected or to add more context to logs gathered from the [`cockroach debug zip`](cockroach-debug-zip.html) command.
+
+The node that `cockroach debug job-trace` is run against will communicate to all nodes in the cluster in order to retrieve the trace payloads. This will deliver a zip file that contains [trace files](#files) for all the nodes participating in the execution of the job. The files hold information on the executing job's [trace spans](show-trace.html#trace-description), which describe the sub-operations being performed. Specifically, these files will contain the spans that have not yet completed and are associated with the execution of that particular job. Using this command for a job that is not currently running will result in an empty zip file.
+
+## Synopsis
+
+~~~ shell
+$ cockroach debug job-trace <job_id> --url=<connection string> {flags}
+~~~
+
+See [`SHOW JOBS`](show-jobs.html#show-jobs) for details on capturing a `job_id`.
```

Comment:
oh nice - ya would also link to this from the first mention of job above too tho